infra/cifuzz.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module used by CI tools in order to interact with fuzzers.
+This module helps CI tools do the following
+  1. build fuzzers
+  2. run fuzzers
+Eventually it will be used to help CI tools determine which fuzzers to run.
+"""
+
+import argparse
+import os
+import tempfile
+
+import build_specified_commit
+import repo_manager
+import utils
+
+
+def main():
+  """Connects Fuzzers with CI tools."""
+  parser = argparse.ArgumentParser(
+      description='Help CI tools manage specific fuzzers.')
+
+  subparsers = parser.add_subparsers(dest='command')
+  build_fuzzer_parser = subparsers.add_parser(
+      'build_fuzzers', help='Build an OSS-Fuzz projects fuzzers.')
+  build_fuzzer_parser.add_argument('project_name')
+  build_fuzzer_parser.add_argument('repo_name')
+  build_fuzzer_parser.add_argument('commit_sha')
+
+  run_fuzzer_parser = subparsers.add_parser(
+      'run_fuzzers', help='Run an OSS-Fuzz projects fuzzers.')
+  run_fuzzer_parser.add_argument('project_name')
+  args = parser.parse_args()
+
+  # Change to oss-fuzz main directory so helper.py runs correctly.
+  if os.getcwd() != utils.OSS_FUZZ_HOME:
+    os.chdir(utils.OSS_FUZZ_HOME)
+  if args.command == 'build_fuzzers':
```

Comment:
Put a blank line here since the comment is referring to 48-49.

Reply:
Done.